dotnet/src/SemanticKernel.Abstractions/Reliability/NullHttpHandler.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Net.Http;
+
+namespace Microsoft.SemanticKernel.Reliability;
+
+/// <summary>
+/// A http retry handler that does not retry.
```

Comment:
Is it a retry handler or just an http handler? I think we are expecting callers to do other http handling actions here, is that right? Makes me wonder if `Reliability` is the right namespace any longer.

Reply:
That's a good point, might need to move this out of `Resiliency` namespace as this is a more general purpose abstraction.